venia-integration-tests/src/assertions/ProductPage/index.js

```diff
@@ -0,0 +1,9 @@
+import { productPageSelectedWishlistButton } from '../../fields/productPage';
+
+/**
+ * Utility function to add product to wishlist from product page
```

Comment:
I don't think this comment is correct. This asserts that the product's wishlist indicator button is filled in / exists, right?

Reply:
yeah, copy/paste error :P